dev/com.ibm.ws.javaee.ddmodel/resources/com/ibm/ws/javaee/internal/ddmodel/resources/DDModelMessages.nlsprops

```diff
@@ -34,9 +34,22 @@ required.attribute.missing=CWWKC2251E: The {2} element is missing the required {
 required.attribute.missing.explanation=A required attribute is missing from the specified element.
 required.attribute.missing.useraction=Correct the deployment descriptor by adding the missing attribute.
 
-invalid.root.element=CWWKC2252E: Invalid root local name {2} in the {0} deployment descriptor on line {1}.
-invalid.root.element.explanation=The root element is incorrect.
-invalid.root.element.useraction=Validate the xml file to make sure the root element is correct.
+# REMOVED:
+# invalid.root.element=CWWKC2252E: Invalid root local name {2} in the {0} deployment descriptor on line {1}.
+# invalid.root.element.explanation=The root element is incorrect.
+# invalid.root.element.useraction=Validate the xml file to make sure the root element is correct.
+
+# REPLACED BY:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the configuration file
+#    {2} is the actual root element name
+#    {3} is the expected root element name
+# -----------------------------------------------------------------------
+unexpected.root.element=CWWKC2252E: The deployment descriptor {0}, at line {1}, has root element {2}, but requires root element {3}. 
```

Comment:
Change "deployment descriptor {0}" to "{0} deployment descriptor"

Reply:
Updated.